video/out/x11_common.c

```diff
@@ -1606,10 +1609,16 @@ static void vo_x11_map_window(struct vo *vo, struct mp_rect rc)
     }
 
     if (x11->opts->all_workspaces || x11->opts->geometry.ws > 0) {
-        long v = x11->opts->all_workspaces
-                ? 0xFFFFFFFF : x11->opts->geometry.ws - 1;
-        XChangeProperty(x11->display, x11->window, XA(x11, _NET_WM_DESKTOP),
-                        XA_CARDINAL, 32, PropModeReplace, (unsigned char *)&v, 1);
+        if (x11->wm_type & vo_wm_STICKY) {
+            Atom state = XA(x11, _NET_WM_STATE_STICKY);
+            XChangeProperty(x11->display, x11->window, XA(x11, _NET_WM_STATE), XA_ATOM,
+                            32, PropModeAppend, (unsigned char *)&state, 1);
```

Comment:
not that it matters but `PropModeReplace` maybe?

Reply:
agree